datafusion/sqllogictest/test_files/explain_tree.slt

```diff
@@ -725,7 +725,12 @@ physical_plan
 32)└─────────────┬─────────────┘
 33)┌─────────────┴─────────────┐
 34)│       LazyMemoryExec      │
-35)└───────────────────────────┘
+35)│    --------------------   │
+36)│     batch_generators:     │
+37)│ generate_series: start=1, │
+38)│    end=1000, batch_size   │
```

Comment:
my recommendation is to avoid printing the batch size here if possible

And maybe we could add one line per generator 🤔